Running on windows: "panic: exec: "docker": executable file not found in $PATH"

HI,

I am trying to run dive using the provided instructions for windows:

`docker run --rm -it -v /var/run/docker.sock:/var/run/docker.sock wagoodman/dive:latest <some:image>`

This is the output:
```
Analyzing Image
panic: exec: "docker": executable file not found in $PATH

goroutine 1 [running]:
github.com/wagoodman/dive/utils.DiscoverDockerVersion(0x94e220, 0x0)
        /home/wagoodman/go/src/github.com/wagoodman/dive/utils/docker.go:42 +0x18d
github.com/wagoodman/dive/image.InitializeData(0x7ffcea2b3f34, 0x4f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/wagoodman/go/src/github.com/wagoodman/dive/image/image.go:202 +0xf2
github.com/wagoodman/dive/cmd.analyze(0xc651a0, 0xc4202b1be0, 0x1, 0x1)
        /home/wagoodman/go/src/github.com/wagoodman/dive/cmd/analyze.go:36 +0x119
github.com/spf13/cobra.(*Command).execute(0xc651a0, 0xc420094010, 0x1, 0x1, 0xc651a0, 0xc420094010)
        /home/wagoodman/go/src/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/spf13/cobra.(*Command).ExecuteC(0xc651a0, 0x2, 0x7f88f39ef6c8, 0x0)
        /home/wagoodman/go/src/github.com/spf13/cobra/command.go:852 +0x30a
github.com/spf13/cobra.(*Command).Execute(0xc651a0, 0xc4201cdf78, 0xc4201cdf58)
        /home/wagoodman/go/src/github.com/spf13/cobra/command.go:800 +0x2b
github.com/wagoodman/dive/cmd.Execute()
        /home/wagoodman/go/src/github.com/wagoodman/dive/cmd/root.go:30 +0x2d
main.main()
        /home/wagoodman/go/src/github.com/wagoodman/dive/main.go:38 +0xad
```

Is there a workaround for windows?

--- a/README.md
+++ b/README.md
@@ -91,9 +91,13 @@ docker run --rm -it \
     -v $(which docker):/bin/docker \
     wagoodman/dive:latest <dive arguments...>
 ```
-For docker in windows (does not support pulling images yet):
+
+Docker for Windows (showing PowerShell compatible line breaks; collapse to a single line for Command Prompt compatibility)
 ```bash
-docker run --rm -it -v //var/run/docker.sock:/var/run/docker.sock wagoodman/dive:latest <dive arguments...>
+docker run --rm -it `
+    -v /var/run/docker.sock:/var/run/docker.sock `
+    -v /usr/local/bin/docker:/bin/docker `
+    wagoodman/dive:latest <dive arguments...>
 ```
 
 ## KeyBindings

